src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -1380,6 +1393,25 @@ public void importPreferences(Path file) throws JabRefException {
         }
     }
 
+    public void exportTheme(Path targetFile, String theme) throws JabRefException {
+        try (OutputStream os = Files.newOutputStream(targetFile)) {
+            if (theme.equals(ThemeLoader.MAIN_CSS) || theme.equals(ThemeLoader.DARK_CSS)) {
+                Path path = new File(JabRefFrame.class.getResource(theme).toURI()).toPath();
+                writeThemeToFile(path, os);
+            } else {
+                Path path = new File(theme).toPath();
+                writeThemeToFile(path, os);
+            }
+        } catch (IOException | URISyntaxException ex) {
+            throw new JabRefException("Could not export theme", Localization.lang("Could not export theme"),
+                    ex);
+        }
+    }
+
+    private void writeThemeToFile(Path path, OutputStream os) throws IOException {
+        Files.copy(path, os);
+    }
```

Comment:
In this case, writeThemeToFile is just an alias for Files.copy with the exact arguments. Just inline it. "exportTheme" should be clear enough.